libraries/mbed/targets/hal/TARGET_Freescale/TARGET_KPSDK_MCUS/serial_api.c

```diff
@@ -62,16 +62,20 @@ void serial_init(serial_t *obj, PinName tx, PinName rx) {
     UART_HAL_SetStopBitCount(uart_addrs[obj->index], kUartOneStopBit);
     #endif
     UART_HAL_SetBitCountPerChar(uart_addrs[obj->index], kUart8BitsPerChar);
-    UART_HAL_EnableTransmitter(uart_addrs[obj->index]);
-    UART_HAL_EnableReceiver(uart_addrs[obj->index]);
+    UART_HAL_DisableTransmitter(uart_addrs[obj->index]);
+    UART_HAL_DisableReceiver(uart_addrs[obj->index]);
 
     pinmap_pinout(tx, PinMap_UART_TX);
     pinmap_pinout(rx, PinMap_UART_RX);
 
     if (tx != NC) {
+        UART_HAL_FlushTxFifo(uart_addrs[obj->index]);
```

Comment:
Flush the TX FIFO fixes and issue where the K64F drops the first character it sends over serial. ex. I want to send "Start" but all I receive is "tart". This fixed the issue for me.

I can add flushing of the RX FIFO too. I think I was a little hesitant because I didn't want to lose any received data that might be useful. Thoughts?